packages/documentation-site/patternfly-docs/content/design-guidelines/components/pagination/pagination.md

```diff
@@ -6,65 +6,86 @@ section: components
 import '../components.css';
 
 ## Usage
-Add pagination above and below content views so that users can easily navigate as they read through listed items on a page.
+Pagination allows users to conveniently navigate through content-dense components or page elements.
 
 <div class="ws-docs-content-img">
 ![a full page demo with a table and a pagination component within a toolbar.](./img/pagination-elements.svg)
 </div>
 
-1. **Top pagination:** Always right-align pagination in the top toolbar, above content views.
+1. **Top pagination:** Placed above the component it controls and aligned to the right.
+
+2. **Bottom pagination:** Placed below the component it controls. Right-aligned for desktop views and center-aligned for mobile. If you want bottom pagination to be sticky so that it remains in view as users scroll through long content, place it directly after the component it controls and refer to the [React sticky pagination](/components/pagination#sticky) or [HTML sticky pagination](/components/pagination/html#bottom-sticky) examples for implementation guidance.
```

Comment:
FWIW it's center-aligned and should be sticky on mobile by default, so the pagination should always be placed directly after the thing it applies to so the sticky positioning works on mobile. You probably know this but the sticky variants make it sticky on desktop, too.